packages/distributor-oracle/src/server.ts

```diff
@@ -165,7 +165,7 @@ export class OracleServer {
       "/v1/tuktuk/wallet/:wallet",
       this.tuktukWalletHandler.bind(this)
     );
-    this.app.post("/v1/sign/:keyToAssetKey", this.signHandler.bind(this));
+    this.app.post("/v1/sign", this.signHandler.bind(this));
```

Comment:
Is it going to break anything public to change this API? We could keep the old version if so.